instagram/src/components/PostContainer/PostContatiner.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import CommentSection from '../CommentSection/CommentSection';
+
+const PostContainer = props => {
+    return( 
+    <div>    
+        
+        {props.dataProps.map(point => 
+        <div>
+            <img src={point.imageUrl} />
+
+            <CommentSection 
+                comment={point.comment} 
+                key={point.username} 
```

Comment:
the `key` actually needs to be on the outermost element being returned by the map method -- so in this case, the classless `div` a few lines up